Add sample for format control sequences

KornelH opened the pull request:
Add an example file for visual inspection of syntax highlight of `io:fwrite` and `io:fread` format control sequences.

It is a follow up for pull request #8.

KornelH committed:
Add sample for format control sequences

Add an example file for visual inspection of syntax highlight of
`io:fwrite` and `io:fread` format control sequences.

It is a follow up for pull request #8.

robertoaloi commented:
@KornelH I guess we want to add a snapshot test for this based on @garazdawi 's PR #9

KornelH committed:
Make test tasks compatible to Windows

For test task execution (`tasks.json`) the default shell on Windows is
PowerShell and conditional execution operator `&&` is not available in
PowerShell prior version 7. In PowerShell, the closest thing we can do
is `(command1 ...) -and (command2 ...)`. It has the same logic, but the
output text from the commands are lost. So, let's use Windows Command
Line (`cmd.exe`) instead that has conditional execution operator `&&`.

Alternative solution could be to extract test commands into scripts, but
as it's only two very short commands yet, it feels simpler to keep them
in `tasks.json`.

From other hand, `npm` uses Windows Command Line to run scripts from
`package.json`, that is just fine for us.

KornelH commented:
That's right. I've checked how this grammar test thingy works. It quite simple to use. Cool! 😎 So, I added the snapshot files.

In addition I updated the test commands as on Windows it didn't work. I haven't tested it on Linux, but according the above message ("All check have passed") it seems it works properly on Linux.

robertoaloi commented:
Thanks!
